examples/systemcalls/systemcalls.c

```diff
@@ -93,7 +124,38 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
-    va_end(args);
 
-    return true;
+    fflush(stdout);
```

Comment:
@bharathvarmaCU18 think about how you could avoid duplication with this function and the one above.